examples/kubernetes-istio/authn.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: security.istio.io/v1beta1
+kind: PeerAuthentication
+metadata:
+  name: default
+spec:
+  mtls:
+    mode: DISABLE
```

Comment:
A few of these yamls are missing trailing newlines; make sure your editor adds them! I'll push a quick fixup and merge.